dns/operationcode.org/variables.tf

```diff
@@ -6,4 +6,9 @@ variable "hosted-zone" {
 variable "k8s-cluster-ingress" {
   description = "Load balancer URL for the Kubernetes ingress"
   type = "string"
+}
+
+variable "pybot-lb-ingress" {
```

Comment:
Fair to put this in a var, but not particularly necessary unless you intend to reuse it multiple times. The k8s cluster address is made into a variable in this instance because it's the CNAME record for every externally accessible service in our cluster.

Reply:
It'll be used for the staging environment as well, I just wanted to get this up to date first.

Also, just figured it kept everything cleaner.